cranelift/codegen/src/machinst/buffer.rs

```diff
@@ -1286,6 +1313,18 @@ pub struct MachSrcLoc {
     pub loc: SourceLoc,
 }
 
+/// Record of stackmap metadata: stack offsets containing references.
+#[derive(Clone, Debug)]
+pub struct MachStackMap {
+    /// The code offset at which this stackmap applies.
+    pub offset: CodeOffset,
+    /// The code offset at the *end* of the instruction at which this stackmap
```

Comment:
Errr, `*end*` is too vague.  Please specify exactly.  Do you mean: the code offset for the last (highest-addressed) byte of the instruction ..

Reply:
Updated comment, thanks!